test/e2e/errorcondition_test.go

```diff
@@ -40,7 +40,9 @@ const (
 // for the container image missing scenario.
 
 func TestContainerErrorMsg(t *testing.T) {
-	//t.Skip("Skipping until https://github.com/knative/serving/issues/1240 is closed")
+	if strings.HasSuffix(strings.Split(test.Flags.DockerRepo, "/")[0], ".local") {
```

Comment:
The only reason I didn't hardcode it is because `dev.local` is apparently used as well. At least, it's in the default value of `registriesSkippingTagResolving` in the `config-controller` ConfigMap.

Reply:
Thanks for the pointer. Fine to keep it as is of course.